backend/api/grants/forms.py

```diff
@@ -23,13 +23,23 @@ def clean(self):
         if not conference.is_grants_open:
             raise exceptions.ValidationError(_("The grants form is now closed!"))
 
+        try:
+            Grant.objects.get(user_id=self.context.request.user.id)
+            raise exceptions.ValidationError(_("Grant already submitted!"))
+        except Grant.DoesNotExist:
+            pass
+
```

Comment:
```
if Grant.objects.filter(user_id=self.context.request.user.id).exists()
```

this way django doesn't do an a select query fetching data/creating the object just to throw it away